pages/common/wget.md

```diff
@@ -5,24 +5,28 @@
 
 - Download the contents of an URL to a file:
 
-`wget -O {{filename}} {{url}}`
+`wget {{https://example.com}}`
 
-- Limit download speed:
+- Download a single web page and all its associated resources:
 
-`wget --limit-rate={{200k}} {{url}}`
+`wget --page-requisites --convert-links {{https://example.com/somepage.html}}`
 
-- Continue an incomplete download:
+- Download a full website, with 3-second intervals between requests:
```

Comment:
`with a 3 second delay`?

Reply:
Just that? It may not be sufficiently clear that the delay is between each request rather than, say, before starting the entire process. That's an unlikely use case, sure but I'd prefer avoiding the ambiguity if we can.

Reply:
Ah, true. Let's leave it as is then :smiley_cat: 

I meant to send this as a general comment, not a change request, oops :P